src/OpenTelemetry.Instrumentation.SqlClient/.publicApi/PublicAPI.Unshipped.txt

```diff
@@ -8,8 +8,6 @@ OpenTelemetry.Instrumentation.SqlClient.SqlClientTraceInstrumentationOptions.Rec
 OpenTelemetry.Instrumentation.SqlClient.SqlClientTraceInstrumentationOptions.RecordException.set -> void
 OpenTelemetry.Instrumentation.SqlClient.SqlClientTraceInstrumentationOptions.SetDbQueryParameters.get -> bool
 OpenTelemetry.Instrumentation.SqlClient.SqlClientTraceInstrumentationOptions.SetDbQueryParameters.set -> void
```

Comment:
Sorry, but I missed the PR this was introduced in. The `[db.query.parameter.<key>](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/registry/attributes/db.md)` attribute is not stable and still considered in development. This means it could be renamed or perhaps even removed altogether.

Because of this, I do not think we should expose this configuration publicly in prep for the stable release. I'd be ok gating it behind an experimental environment variable or something.